vmware_exporter/vmware_exporter.py

```diff
@@ -464,7 +471,8 @@ def _vmware_get_vms(self, content, metrics, inventory):
                         snapshot['timestamp_seconds'],
                     )
 
-        return result
+        yield d
```

Comment:
Is there a better name then "d" that we could use? Maybe vm_metric_data?

Reply:
I was hesitant to call it something that implied it was a variable holding collected data, as when the deferred / promise is resolved its value is just None. I've gone with `vm_perf_deferred`.